cmd/jaeger/internal/integration/es_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
+)
+
+const (
+	host          = "0.0.0.0"
+	queryPort     = "9200"
+	queryHostPort = host + ":" + queryPort
+	queryURL      = "http://" + queryHostPort
+)
+
+type ESStorageIntegration struct {
+	E2EStorageIntegration
+	esClient *integration.EsClient
+}
+
+func (s *ESStorageIntegration) initializeES(t *testing.T) {
+	s.esClient = integration.StartEsClient(t, queryURL)
+	s.CleanUp = func(t *testing.T) {
+		s.esClient.DeleteAllIndixes(t)
+	}
+	s.esClient.DeleteAllIndixes(t)
```

Comment:
let's implement this in Purge method of the ES Factory and rely on cleaner extension